package/model-pytorch-rnnt/README.md

```diff
@@ -0,0 +1,7 @@
+# RNN-T package
+
+```
+$ ck clean env --tags=model,pytorch,rnnt -f
+$ ck install package:model-pytorch-rnnt
```

Comment:
Why not use `--tags` as above.

Reply:
Done - a cut and paste from the previous readme, if I'm honest.